### PR TITLE
Flatpages and Profile changes

### PR DESCRIPTION
**Pull request by comzeradd**

### CODE REVIEW

**Comment by comzeradd** [comment]
@johngian r?

File: feedthefox/base/templates/jinja2/profile.html [hunk 1]
```diff
@@ -45,17 +45,15 @@
               <tr>
                 <td>
                   <span class="glyphicon glyphicon-phone" aria-hidden="true"></span>
-                  {{ device_info.device.model }}
+                  <a href="{{ url('device', manufacturer=device_info.device.manufacturer, model=device_info.device.model) }}">{{ device_info.device }}</a>
                 </td>
                 <td class="profile-imei hidden-sm hidden-xs" title="Your IMEI number is visible only to you">
-                  <span class="glyphicon glyphicon-eye-open" aria-hidden="true"></span>
-                  {{ device_info.imei }}
+                  {% if device_info.imei %}
```
Comment (johngian) [inline feedthefox/base/templates/jinja2/profile.html:51]: Is there any chance we have a device without imei?

**Comment by johngian** [comment]
Code looks OK. Let's wait for @ppapadeas answer on `imei` (this would also require a change to the model).

**Comment by johngian** [comment]
r+wc

File: feedthefox/base/templates/jinja2/profile.html (hunk 1 above)
Comment (johngian) [inline feedthefox/base/templates/jinja2/profile.html:51]: If not let's omit `if/else`.

Comment (comzeradd) [inline feedthefox/base/templates/jinja2/profile.html:51]: Not all users required to enter IMEI, so let's adjust the backend to this.

**Pull request merged**